synology/start_tunnel_to_server.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+zabbix_server_port="${1}"
+priv_key="${2}"
+
+if [ -z "${zabbix_server_port}" ]  || [ -z "${priv_key}" ]; then
+  echo "Usage:"
+  echo "  ${0} <zabbix_server_port> <priv_key>"
+  exit 1
+fi
+
+if [ -z "${tmp_dir}" ]; then
+  tmp_dir=$(mktemp -d)
+fi
+
+echo "Waiting for autossh to become available..."
+
+while [ ! -x /usr/local/bin/autossh ]; do
+  sleep 10
+done
+
+echo "Starting tunnel to Server..."
+
+/usr/local/bin/autossh \
```

Comment:
This command is not complete, look at [this command][remote_tunnel] to see how to do it.
You will need to specify quite a couple more options inside the proxy command.

[remote_tunnel]: https://github.com/MSF-OCB/remote-tunnels/blob/548f9f41b2b290c54d397cdef68495af5a52a1c9/remote/create_tunnel.sh#L180